tools/internal_ci/linux/grpc_e2e_performance_v2.sh

```diff
@@ -19,5 +19,28 @@ cd $(dirname $0)/../../..
 
 source tools/internal_ci/helper_scripts/prepare_build_linux_rc
 
+# This is to insure we can push and pull images from gcr.io, we do not
+# necessarily need it to run load tests, but would need it when we employ
+# pre-built images in the optimization.
+gcloud auth configure-docker
+
+# Connect to benchmarks-prod cluster
+gcloud config set project grpc-testing
+gcloud container clusters get-credentials benchmarks-prod \
+    --zone us-central1-b --project grpc-testing
+
+# This step subject to change, just easier to get one file to run a single test
+wget https://raw.githubusercontent.com/grpc/test-infra/master/config/samples/go_example_loadtest.yaml
+
+# The original version of the client is a bit old, update to a newer version.
+kubectl version --client
+curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
```

Comment:
nit: I'd just hardcode the last stable version available now. You're probably not very dependent on kubectl version so the extra complexity is unnecessary.

Reply:
I found the latest released version is v1.21.0 (https://kubernetes.io/docs/setup/release/notes/#), hardcoded that version in the file. Thank you for reviewing.